src/etc/inc/openvpn.inc

```diff
@@ -447,6 +447,16 @@ function openvpn_get_digestlist() {
 	return $digests;
 }
 
+function openvpn_get_keydirlist() {
+	$keydirs = array(
+		'-1' => gettext('Use default keydir direction'),
+		'2' => gettext('No keydir direction'),
+		'0'  => gettext('Use keydir direction 0'),
+		'1'  => gettext('Use keydir direction 1'),
```

Comment:
See above, I'd keep the default option and leave these are simply "0" and "1". We may default them a certain way but the other way is also valid so long as it is set in a complementary way, so I wouldn't change the text to indicate anything other than 0/1 since it is ultimate up to the user.